serverless.yml

```diff
@@ -156,5 +156,5 @@ custom:
         - eu-west-2
         - amazonaws.com
   aliases:
-    staging: staging.frontdoor-snapshot.hackney.gov.uk
+    staging: frontdoor-snapshot-staging.hackney.gov.uk
```

Comment:
I like the other page better, but there was already a wildcard cert for *.hackney.gov.uk (which doesn't work with multiple subdomain levels).